File: storydoc-frontend/src/app/api/models/screen-design-dto.ts
/* tslint:disable */
/* eslint-disable */
import { SdContainerDto } from './sd-container-dto';
import { ScreenDesignCoordinate } from './screen-design-coordinate';
import { StoryDocSummaryDto } from './story-doc-summary-dto';
export interface ScreenDesignDto {
  coordinate?: ScreenDesignCoordinate;
  name?: string;
  rootContainer?: SdContainerDto;
  storyDocSummaryDTO?: StoryDocSummaryDto;
}

File: storydoc-backend/src/main/java/io/storydoc/server/ui/app/screendesign/ScreenDesignDTO.java
package io.storydoc.server.ui.app.screendesign;

import io.storydoc.server.storydoc.app.dto.StoryDocSummaryDTO;
import io.storydoc.server.ui.domain.screendesign.ScreenDesignCoordinate;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ScreenDesignDTO {
    private ScreenDesignCoordinate coordinate;
    private StoryDocSummaryDTO storyDocSummaryDTO;
    private String name;
    private SDContainerDTO rootContainer;
}
